Makefile

```diff
@@ -4,7 +4,7 @@ PROJECT = $(subst @,,$(notdir $(subst /workspace,,$(PROJECT_PATH))))
 
 export PROJECT
 
-TMP = tmp/capybara tmp/junit tmp/codeclimate coverage log/test.searchd.log
+TMP = tmp/capybara tmp/junit coverage log/test.searchd.log
```

Comment:
Is it complicated to keep coverage report in `tmp/`? Just not to pollute the root project dir.

Reply:
Do you mean if we can move current coverage files to tmp/?

Reply:
This is what I meant but probably better to keep then in the standard location.

Reply:
Ignore this.